activesupport/lib/active_support/core_ext/date/conversions.rb

```diff
@@ -80,6 +80,7 @@ def readable_inspect
   #
   #   date.to_time(:utc)             # => 2007-11-10 00:00:00 UTC
   def to_time(form = :local)
+    raise ArgumentError, "Expected :local or :uts, got #{form}." unless [:local, :utc].include?(form)
```

Comment:
Can we speed this up (use a case; when) or defer it (rescue NoMethodError and re-raise ArgumentError)?